app/code/core/Mage/Adminhtml/controllers/System/Convert/ProfileController.php

```diff
@@ -102,34 +102,25 @@ public function gridAction()
         );
     }
 
-    /**
-     * Profile edit action
-     */
     public function editAction()
     {
         $this->_initProfile();
         $this->loadLayout();
 
+        // Set entered data if was error when we do save
         $profile = Mage::registry('current_convert_profile');
-
-        // set entered data if was error when we do save
         $data = Mage::getSingleton('adminhtml/session')->getConvertProfileData(true);
```

Comment:
The comment should be above `$data` as it is related to it:

```suggestion
        /** @var Mage_Dataflow_Model_Profile $profile */
        $profile = Mage::registry('current_convert_profile');

        // Repopulate user input values in the edit form in case of saving error 
        $data = Mage::getSingleton('adminhtml/session')->getConvertProfileData(true);
```

Reply:
I reformatted it this way because it seems that comment belongs to this whole logical section:
<img width="744" alt="Screenshot 2024-02-09 alle 09 56 10" src="https://github.com/OpenMage/magento-lts/assets/909743/c1ef2f36-5714-4a9b-ab19-8f6a72a2a545">

I can revert it but those variables and that comment kinda belong together :-)